taipy/gui/viselements.json

```diff
@@ -109,6 +109,12 @@
                         "type": "int",
                         "default_value": "5",
                         "doc": "The height of the displayed element if multiline is True."
+                    },
+                    {
+                        "name": "type",
+                        "type": "str",
+                        "default_value": "text",
+                        "doc": "The type of input: text, tel, email ..."
```

Comment:
Is there a list to pick meaningful values from?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#input_types

Reply:
How do users know that?
Oh I know. Let them know.

Reply:
or not ...